client/src/components/ManageAccount/ManageAccount.scss

```diff
@@ -128,8 +131,7 @@
 // Buttons and Forms
 .manage-account-submit-button {
   display: flex;
-  justify-content: space-between;
-  align-items: center;
+  width: fit-content;
```

Comment:
Does this do anything without specifying a content size? 

https://developer.mozilla.org/en-US/docs/Web/CSS/width

Reply:
Believe it defaults to `fit-content(stretch)`
https://developer.mozilla.org/en-US/docs/Web/CSS/fit-content